test/PowerShellEditorServices.Test/Extensions/ExtensionServiceTests.cs

```diff
@@ -206,6 +206,11 @@ public Task CloseFile(string filePath)
         }
 
         public Task SaveFile(string filePath)
+        {
+            return SaveFile(filePath, null);
```

Comment:
Does this mean we could write a test for this :)

Reply:
Do we have tests for the editor method tasks? I thought I'd route through the methods on the mocks, but can happily add to tests.

Reply:
looks like it's just mocks. No actual tests.